.circleci/config.yml

```diff
@@ -54,6 +54,7 @@ commands:
       - run: echo "export const version = '`git describe --always`';" > ./src/version.ts
       - run: npm run injectDeployUrlForAssets --url='//assets.algorea.org<< parameters.path >><< parameters.lang >>/'
       - run: npm run injectDeployUrlForPreloadFonts --url='//assets.algorea.org<< parameters.path >><< parameters.lang >>/'
+      - run: npm run injectDeployUrlForBgUrlAssets --url='//assets.algorea.org<< parameters.path >><< parameters.lang >>/'
```

Comment:
why separating these from the other image assets?

Reply:
Fixed